docs/geometry/pick.md

```diff
@@ -1,30 +1,33 @@
-## Pick 定理
+## Pick's theorem
 
-Pick 定理：给定顶点坐标均是整点（或正方形格子点）的简单多边形，皮克定理说明了其面积 ${\displaystyle A}$ 和内部格点数目 ${\displaystyle i}$ 、边上格点数目 ${\displaystyle b}$ 的关系： ${\displaystyle A=i+{\frac {b}{2}}-1}$ 。
+Pick's theorem: Given a simple polygon whose vertex coordinates are all integral points, Pick’s theorem proves the relationship between its area ${\displaystyle A}$ , the number of internal points ${\displaystyle i}$ , and the number of points on the edge ${\displaystyle b}$ : ${\displaystyle A=i+{\frac {b}{2}}-1}$ .
 
-具体证明： [Pick's theorem](https://en.wikipedia.org/wiki/Pick%27s_theorem) 
+Detailed proof: [Pick's theorem](https://en.wikipedia.org/wiki/Pick%27s_theorem) 
 
-它有以下推广：
+It has the following generalizations:
 
-- 取格点的组成图形的面积为一单位。在平行四边形格点，皮克定理依然成立。套用于任意三角形格点，皮克定理则是 ${\displaystyle A=2 \times i+b-2}$ 。
-- 对于非简单的多边形 ${\displaystyle P}$ ，皮克定理 ${\displaystyle A=i+{\frac {b}{2}}-\chi (P)}$ ，其中 ${\displaystyle \chi (P)}$ 表示 ${\displaystyle P}$ 的 **欧拉特征数** 。
-- 高维推广：Ehrhart 多项式
-- 皮克定理和 **欧拉公式** （ ${\displaystyle V-E+F=2}$ ）等价。
+- The area of the graph composed of grid points is one unit. Pick's theorem still stands for parallelogram grid. For any triangle lattice, Pick's theorem is ${\displaystyle A=2 \times i+b-2}$ .
 
-## 一道例题 (POJ 1265)
+- For non-simple polygons ${\displaystyle P}$ , Pick’s theorem ${\displaystyle A=i+{\frac {b}{2}}-\chi (P)}$ , where ${\displaystyle \chi ( P))$ represents the **Euler characteristic** of $(\displaystyle P)$ .
 
-### 题目大意
+- Higher-dimensional: Ehrhart polynomials.
+- Pick's theorem is equivalent to Euler's formula ( ${\displaystyle V-E+F=2}$ ).
 
-给一个平面上的简单多边形，求边上的点，多边形内的点，多边形面积。
+## Sample problem (POJ 1265)
+
+### Topic
+
+Given a simple polygon on a plane, find the points on the side, inside the polygon, and the area of the polygon.
 
 ### Solution
 
-这道题目其实用了以下三个知识：
+This topic includes the following three key points:
+
+- For a line segment with an integral point as vertex, the number of points covered is $\gcd(\textit{dx},\textit{dy})$ , where $\textit{dx},\textit{dy}$ are the number of points occupied by the line segment horizontally and vertically. If $\textit{dx}$ or $\textit{dy}$ is $0$ , the number of points covered is $\textit{dy}$ **or** $\textit{dx}$ .
+- Pick's theorem: The area of a simple polygon with integral points as vertices on the plane = the number of points on the edge/2 + the number of internal points + 1.
+- The area of any polygon is equal to finding the sum of the cross product of the vector composed of two adjacent points and the origin in order (this can also be obtained by clockwise definite integral).
```

Comment:
```suggestion
- The number of points covered by the line segment with the integral point as the vertex is $\gcd(\textit{dx},\textit{dy})$ , where $\textit{dx} and \textit{dy}$ are respectively the number of points covered by line horizontally and vertically. If $\textit{dx}$ or $\textit{dy}$ is $0$ , the number of points covered is $\textit{dy}$ **or** $\textit{dx}$ .
- Pick theorem: The area of a simple polygon with integral points as its vertices on the plane = the number of points on the side/2 + the number of internal points + 1.
- The area of any polygon is equal to the sum of the cross product of the vector composed of two adjacent points and the origin in order (this can also be obtained by clockwise definite integral).
```